ephios/plugins/basesignup/signup/common.py

```diff
@@ -114,7 +114,9 @@ def get_participation_display(self):
                 ", ".join(
                     participant.qualifications.filter(
                         category__in=relevant_qualification_categories
-                    ).values_list("title", flat=True)
+                    )
+                    .order_by("title")
```

Comment:
Did you try ordering by Category? Might be sensible as well...

Reply:
good point

Reply:
Now we can't change the ordering in the relevent-qualification settings, sadly.